imdraw/imdraw.go

```diff
@@ -51,9 +51,11 @@ type IMDraw struct {
 	Precision int
```

Comment:
There is no problem for 10k slices being in the pool. There will always be at most a constant number of slices in the pool.